.gitignore

```diff
@@ -6,6 +6,7 @@ dist-es6/*
 node_modules/
 coverage/
 .nyc_output/
+.npmrc
```

Comment:
Why?

Reply:
This was the way I found to have npm use the public registry for this project while continuing to use internal npm for other projects. I place an `.npmrc` file where there's a `project.json` with the following contents:

```
registry=https://registry.yarnpkg.com
```